operator/src/main/java/oracle/kubernetes/operator/helpers/WebhookHelper.java

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2022, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator.helpers;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+import io.kubernetes.client.openapi.models.AdmissionregistrationV1ServiceReference;
+import io.kubernetes.client.openapi.models.AdmissionregistrationV1WebhookClientConfig;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1RuleWithOperations;
+import io.kubernetes.client.openapi.models.V1ValidatingWebhook;
+import io.kubernetes.client.openapi.models.V1ValidatingWebhookConfiguration;
+import oracle.kubernetes.common.logging.MessageKeys;
+import oracle.kubernetes.operator.calls.CallResponse;
+import oracle.kubernetes.operator.logging.LoggingFacade;
+import oracle.kubernetes.operator.logging.LoggingFactory;
+import oracle.kubernetes.operator.steps.DefaultResponseStep;
+import oracle.kubernetes.operator.utils.Certificates;
+import oracle.kubernetes.operator.work.NextAction;
+import oracle.kubernetes.operator.work.Packet;
+import oracle.kubernetes.operator.work.Step;
+import org.apache.commons.codec.binary.Base64;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import static oracle.kubernetes.common.logging.MessageKeys.VALIDATING_WEBHOOK_CONFIGURATION_CREATED;
+import static oracle.kubernetes.operator.LabelConstants.CREATEDBYOPERATOR_LABEL;
+import static oracle.kubernetes.operator.helpers.NamespaceHelper.getWebhookNamespace;
+import static oracle.kubernetes.operator.rest.RestConfigImpl.CONVERSION_WEBHOOK_HTTPS_PORT;
+import static oracle.kubernetes.operator.utils.SelfSignedCertUtils.WEBLOGIC_OPERATOR_WEBHOOK_SVC;
+
+public class WebhookHelper {
+  private static final LoggingFacade LOGGER = LoggingFactory.getLogger("Webhook", "Operator");
+  public static final String VALIDATING_WEBHOOK_NAME = "weblogic.validating.webhook";
+  public static final String VALIDATING_WEBHOOK_PATH = "/admission";
+  public static final String APP_GROUP = "weblogic.oracle";
+  public static final String API_VERSION = "v9";
```

Comment:
Reuse the constants from KubernetesConstants so that we only have one place to update.

Reply:
done. thanks.